test/videojs-contrib-hls.test.js

```diff
@@ -2628,6 +2629,74 @@ QUnit.test('cleans up the buffer based on currentTime when loading a live segmen
   assert.deepEqual(removes[0], [0, 80 - 30], 'remove called with the right range');
 });
 
+QUnit.test('cleans up the buffer based on currentTime when loading a live segment if' +
```

Comment:
This test seems pretty complicated to set up. Maybe it would be better if we could make the trim function call another function with the right passed properties to get the trim range, so that that function can be tested in isolation.